### PR TITLE
Further tweaks to the massive legacy views removal PR

### PR DESCRIPTION
**Pull request by borkweb**
These are additional tweaks / adjustments to the deprecation PR #3760

### CODE REVIEW

File: src/functions/template-tags/general.php [hunk 1]
```diff
@@ -1307,6 +1300,26 @@ function tribe_events_tab_index() {
...
+	 *
+	 * @return bool
+	 **/
+	function tribe_events_is_view_enabled( $view ) {
+		$manager = tribe( \Tribe\Events\Views\V2\Manager::class );
+		$public_views = $manager->get_publicly_visible_views();
+
+		$enabled = isset( $public_views[ $view ] );
+
+		return apply_filters( 'tribe_events_is_view_enabled', $enabled, $view, array_keys( $public_views ) );
```
Comment (bordoni) [inline src/functions/template-tags/general.php:1319]: Docblock

File: src/functions/template-tags/day.php [hunk 2]
```diff
@@ -1,51 +1,38 @@
...
- * Get the previous day's date
- *
- * @category Events
- *
- * @return string
- */
-function tribe_get_previous_day_date( ) {
-	return null;
-}
+		return apply_filters( 'tribe_get_day_link', $tribe_ecp->getLink( 'day', $date ), $date );
```
Comment (bordoni) [inline src/functions/template-tags/day.php:36]: This is the V1  way of getting a link `tribe_events_get_url`

**Review by bordoni — CHANGES_REQUESTED** [review]
Two small changes. I mentioned that particular doc block because it felt like one that deserves explanation, but feel free to ignore.

File: src/functions/template-tags/day.php (hunk 2 above)
Comment (borkweb) [reply src/functions/template-tags/day.php:36]: Update it to the v2 way and push those changes.

**Pull request merged**